com.tibia.Tibia.yml

```diff
@@ -0,0 +1,63 @@
+id: com.tibia.Tibia
+branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: tibia-wrapper
+tags:
+  - proprietary
+
+finish-args:
+  - --device=dri # OpenGL rendering
+  - --filesystem=xdg-data/tibia:create
+  - --share=ipc # share IPC namespace with the host (necessary for X11)
+  - --share=network # access the network
+  - --socket=pulseaudio # play sound with PulseAudio
+  - --filesystem=xdg-run/pipewire-0:ro # play sound with PipeWire
+  - --socket=x11 # show windows using X11
+
+modules:
+  - name: tibia
+    buildsystem: simple
+
+    build-commands:
+      - install -d ${FLATPAK_DEST}/share/tibia ${FLATPAK_DEST}/utils
+      - tar xf tibia.tar.gz -C ${FLATPAK_DEST}/share/tibia --strip-components 1
+      - icotool -x --icon ${FLATPAK_DEST}/share/tibia/tibia.ico
+      - install -Dm644 tibia_*_16x16x32.png ${FLATPAK_DEST}/share/icons/hicolor/16x16/apps/${FLATPAK_ID}.png
+      - install -Dm644 tibia_*_32x32x32.png ${FLATPAK_DEST}/share/icons/hicolor/32x32/apps/${FLATPAK_ID}.png
+      - install -Dm644 tibia_*_48x48x32.png ${FLATPAK_DEST}/share/icons/hicolor/48x48/apps/${FLATPAK_ID}.png
+      - install -Dm644 tibia_*_256x256x32.png ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps/${FLATPAK_ID}.png
+      - install -Dm755 -t ${FLATPAK_DEST}/bin tibia-wrapper
+
+    sources:
+      - type: file
+        url: https://static.tibia.com/download/tibia.x64.tar.gz
+        sha256: cdafe67500064036503937714b3b7868f09965b6453287698aeadaef22fec8c2
+        dest-filename: tibia.tar.gz
+
+      - type: script
+        dest-filename: tibia-wrapper
+        commands:
+          - exec /app/share/tibia/Tibia
+
+    cleanup:
+      - /share/tibia/start-tibia-launcher.sh
+      - /share/tibia/tibia.ico
+
+    modules:
+      - modules/icoutils.json
+
+  - name: metadata
+    buildsystem: simple
+
+    build-commands:
+      - install -Dm644 -t ${FLATPAK_DEST}/share/applications com.tibia.Tibia.desktop
+      - install -Dm644 -t ${FLATPAK_DEST}/share/metainfo ${FLATPAK_ID}.metainfo.xml
+
+    sources:
+      - type: file
+        path: com.tibia.Tibia.metainfo.xml
+
+      - type: file
+        path: com.tibia.Tibia.desktop
```

Comment:
Merge this with the main module.

Reply:
All of this metadata module, or just the desktop file?

Reply:
this whole module, metainfo + desktop file